src/inversion/inversion.f90

```diff
@@ -260,7 +310,7 @@ subroutine vorticity_tendency
 
             ! dxi/dt  = dr/dy - dq/dz
             call diffy(r, svorts(:, :, :, 1))
-            call diffz(fp, gp)
+            call central_diffz(fp, gp)
```

Comment:
So central differences work better than the exact differentiation?  That us what produced the best results for EPIC?

Reply:
I mean for PS3D as I assume this is the PS3D code!

Reply:
In PS3D we always used central differencing here.

Reply:
ok!